src/components/manage/Form/Field.jsx

```diff
@@ -83,6 +84,30 @@ const getWidgetPropsFromTaggedValues = (widgetOptions) =>
     ? widgetOptions.frontendOptions.widgetProps
     : null;
 
+/**
+ * Get widget props from patternOptions
+ * @param {object} widgetOptions
+ * @returns {string} Widget component.
+ *
+
+directives.widget(
+    "fieldname",
+    pattern_options={
+        "selectableTypes": [
+            "Image"
+        ]
+    })
+
+ */
+const getWidgetPropsFromPatternOptions = (widgetOptions) =>
+  isArray(widgetOptions?.pattern_options?.selectableTypes) &&
+  intersection(
+    widgetOptions.pattern_options.selectableType,
```

Comment:
we use `frontend_options`, `pattern_options` is for classic. They *must* be different, we cannot assume to use the same. Plone 6 should stick to one, and `frontend_options` is the most sensible one.

So -1 to support this in Volto. Why do you need it?

Reply:
That is how fields are defined currently, so that means the widget will not activate for images if you define them this way.

Reply:
I'll stick to `frontend_options`, I'll open the Pandora's box on this one, before it's too late.